### PR TITLE
Add firstName and lastName attributes to users

### PR DESCRIPTION
**Pull request by kenzshelley**
<h1> this is not that interesting or exciting </h1>

@matthewgrossman @nickcharles

### CODE REVIEW

**Comment by matthewgrossman** [comment]
I'm both interested and excited

**Comment by nickcharles** [comment]
:moneybag: Merge it.

File: test/refresh_db.js [hunk 1]
```diff
@@ -5,20 +5,20 @@ var User = require('../models/user');
 var firebaseRef = new Firebase("https://parq.firebaseio.com");
 
 var users = [
-  {email: "mrgrossm@umich.edu", password: "mrgrossm", id: ""},
-  {email: "nickmorg@umich.edu", password: "nickmorg", id: ""},
-  {email: "kenzshelley@umich.edu", password: "kenzshelley", id: ""}
+  {email: "mrgrossm@umich.edu", password: "mrgrossm", id: "", firstName: "Matt", lastName: "Gross Man"},
+  {email: "nickmorg@umich.edu", password: "nickmorg", id: "", firstName: "Nick", lastName: "Mo"},
+  {email: "kenzshelley@gmail.com", password: "p", id: "", firstName: "Kenz", lastName: "Shelley"}
```
Comment (matthewgrossman) [inline test/refresh_db.js]: you lazy debugger

File: test/refresh_db.js [hunk 2]
```diff
@@ -5,20 +5,20 @@ var User = require('../models/user');
...
 ];
 
 var removeUsers = function() {
   return Promise.all(users.map(function(user) {
-    return firebaseRef.removeUser({email: user.email, password: user.password});
+    return firebaseRef.removeUser({email: user.email, password: user.password, firstName: user.firstName, lastName: user.lastName});
```
Comment (matthewgrossman) [inline test/refresh_db.js]: not sure this is correct. Remove user only takes in an email/password I thought

**Comment by matthewgrossman** [comment]
Not sure if there's a great way to do this, but did you change the fb rules yet? If so, can you just link to the fb rules commit (no need to make a PR for it or anything)

File: test/refresh_db.js (hunk 2 above)
Comment (kenzshelley) [inline test/refresh_db.js]: Oh shit yep, I'll remove that

**Comment by kenzshelley** [comment]
@matthewgrossman I changed them on firebase but have not put the change in the repo yet (oops that's probs the opposite order of how you should do things) -- will add before merging

File: test/refresh_db.js (hunk 1 above)
Comment (kenzshelley) [inline test/refresh_db.js]: :sunglasses:

**Comment by kenzshelley** [comment]
Firebase rules updated in: https://github.com/parq-app/parq-firebase/commit/8c4a6ea8bf77cbc6faa163b204015b1a667faaa3

**Pull request merged**